scenes/prefabs/Dialogue/Dialogue.gd

```diff
@@ -0,0 +1,99 @@
+extends CanvasLayer
+
+export var dialogue_path = "res://res/dialogue/End - auto.csv"
+onready var dialogue_resource = load(dialogue_path)
+
+var _current_event = -1
```

Comment:
Why `-1`? Why not `0`?

Reply:
When `_process_dialogue_event()` is called, `_current_event` is incremented by one at the start, therefore being 0 for the actual process. And the first event is processed when ready. So `_current_event` always shows the latest and current event index.
Why set it to zero and have it increment at the end is because the function doesn't always make it to the end. 
I probably could find a way to refactor to code to allow this. Should I?

Reply:
If you can it'll be great for sure. But it's not a priority I guess